src/libYARP_OS/include/yarp/os/ManagedBytes.h

```diff
@@ -139,6 +139,13 @@ class YARP_OS_API yarp::os::ManagedBytes : public Portable {
         return owned;
     }
 
+    /**
+     * @brief Use this to wrap an external blob of data.
+     * @param extData, pointer to the external data.
+     * @param len, size in bytes of the external blob
+     */
+    void setExternal(const char* extData, size_t len);
```

Comment:
2 comments here.
First of all, this should not be `const char*`, but just char, since the data is not owned, but can be modified. Second, I suggest to use `const Bytes& ext` instead, like in the other constructor, in order to make the interface more coherent

Reply:
I see your point but I tried to be as much coherent as possible to `yarp::sig::Image::setExternal`:
https://github.com/robotology/yarp/blob/29e9108a52713dbe737d9a9c44a62da3adc3ce51/src/libYARP_sig/include/yarp/sig/Image.h#L247-L252 

What is the best option ? :sweat_smile: